tests/commonUtils.sh

```diff
@@ -15,8 +15,8 @@
 #   limitations under the License.
 
 ROOT="$(cd "$(dirname "${BASH_SOURCE[0]}")/.." && pwd)"
-RUNTIME_IMAGE='docker.io/istio/runtime:2017-03-22-21.21.48'
-
+MANAGER_HUB_TAG='docker.io/istio,2017-03-22-21.21.48'
```

Comment:
I agree that instead of hardcoded constants we can get the default hub and tags from istio.VERSION. Note that update_version.sh needs to be fixed now that the `runtime` image has been replaced by `manager` and `proxy` images. I think it would be better for istio.VERSION to just store a single manager hub and tag (`MANAGER = "<hub>,<tag>`) instead of `INIT = ...`, `PROXY= ...`, and `MANAGER = ...`, since they all need to be the same. Can we merge this PR first, and then change the defaults in another PR that also fixes update_version.sh?